mlibrary/patron_account
# [LIBSEARCH-1031] ACCOUNT: Update Settings display to match Figma diagram

# Overview
This pull request was originally focused on [LIBSEARCH-1031](https://mlit.atlassian.net/browse/LIBSEARCH-1031), but the changes were already made. The focus then shifted to CSS and icon changes:

- Used native CSS to style the radio buttons, and removed the need for the icons.
- Used CSS for the arrow icon in the dropdown and removed the need for JavaScript for styling.
- Updated the Material Icons import to only grab the necessary icons instead of the whole font family, improving load time.
- Removed unnecessary CSS files.

Much more could be done, but this was already way out of scope.

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Poke around the site to see if anything is broken.
  - Do the icons load faster?
